docs/release-notes/flink-1.7.md

```diff
@@ -37,6 +120,15 @@ You should only use this feature if you are executing a stateless streaming job.
 In any other cases, it is highly recommended to remove the config option `jobmanager.execution.failover-strategy` from your `flink-conf.yaml` or set it to `"full"`.
```

Comment:
add link to config docs

Reply:
It's no longer documented, thus, there is no link to put here.